asset-transfer-basic/chaincode-java/README.md

```diff
@@ -0,0 +1,164 @@
+
+## Basic asset transfer 
+
+This project demonstrates the use of the Java SDK, running a basic asset transfer contract using the "chaincode as a service" 
+pattern.
+
+
+## Setup 
+
+In this sample we will employ the [Kubernetes Test Network](../../test-network-k8s) to illustrate a scenario of 
+building, running, and debugging chaincode on a development workstation.  
+
+This project is also compatible with the legacy chaincode builder pipeline and the compose based test-network.
+For additional details, see the [End-to-end with the test-network](../../test-network/CHAINCODE_AS_A_SERVICE_TUTORIAL.md#end-to-end-with-the-the-test-network)
+documentation.
+
+## [Quickstart](../../test-network-k8s#quickstart)
+
+```shell
+export PATH=${PWD}/../../test-network-k8s:$PATH 
+
+network kind 
```

Comment:
for common reason, why don't we use network cmd with `./network` instead of `network`?

Reply:
Adding `network` to the PATH makes it a little easier for users to work with other chaincode projects in the fabric-samples project.   I.e.. a user should be able to: 

```
cd fabric-samples

network chaincode deploy asset-transfer-basic/chaincode-java 
``` 

or 

```
cd fabric-samples/test-network-k8s 

network chaincode deploy ../asset-transfer-basic/chaincode-go
```

`./network` is OK but it means users always need to be in the test-network-k8s directory.